### ISSUE TITLE
FileNotFoundError: [WinError 2] 系统找不到指定的文件。

### ISSUE THREAD

**Issue by xlnn** [kingyiusuen/image-to-latex#12]
![image](https://user-images.githubusercontent.com/57608826/143676359-52d90267-c218-4065-af5b-21d9aee91226.png)
这是怎么回事

**Comment by gulabsagevadiya** [comment]
have you found any solution for this error?

**Comment by xlnn** [comment]
> have you found any solution for this error?

no！

**Comment by daescha** [comment]
The problem is you are on windows. It is trying to run a shell script, with the program `sh`. This is a linux program, not available to windows. I would recommend running this script in WSL (windows subsystem for linux) to get around the issue while on a windows machine.

**Comment by KhbAs97** [comment]
> The problem is you are on windows. It is trying to run a shell script, with the program `sh`. This is a linux program, not available to windows. I would recommend running this script in WSL (windows subsystem for linux) to get around the issue while on a windows machine.

Correct, working just fine on Ubuntu - WSL downloaded from Windows Store
